src/xrc-tests/src/mock_responses/exchanges.rs

```diff
@@ -0,0 +1,91 @@
+use serde_json::json;
+use xrc::{usdt_asset, Exchange, EXCHANGES};
+
+use crate::container::{ExchangeResponse, ResponseBody};
+
+/// Build the responses for crytpoexchanges by providing the base and quote asset symbols, the timestamp, and a rate lookup function.
+/// The rate lookup function expects to return rates in a string format (ex. 1.00) per exchange.
```

Comment:
What does "ex. 1.00" mean?
Is that an example for the string format? It looks like a numeric value to me.

Reply:
Yes. It should be a string format. Updated to better reflect that.